content/instructor/workshops/planning-process/index.mdx

```diff
@@ -0,0 +1,122 @@
+---
+date: 2020-04-09
+title: Workshop Planning Process
+showToc: true
+published: true
+categories: ['instructor', 'workshop production']
+---
+
+import DesigningGraphqlSchemasByNikGraf from './designing-graphql-schemas-by-nik-graf'
+import FixCommonGitMistakesByChrisAchard from './fix-common-git-mistakes-by-chris-achard'
+import JourneyWithVueRouterByLaurieBarth from './journey-with-vue-router-by-laurie-barth'
+import ComposableGatsbyThemesByChrisBiscardi from './composable-gatsby-themes-by-chris-biscardi'
+import BuildingUisWithXstateStateMachinesByIsaacMann from './building-uis-with-xstate-state-machines-by-isaac-mann'
+import BuildAnAppWithReactSuspenseByMichaelChan from './build-an-app-with-react-suspense-by-michael-chan'
+
+export default ({ children, ...props }) => <div {...props}>{children}</div>
+
```

Comment:
@tayiorbeii your writing goes here. can also add something between the examples.